core/commands/partials/terraform_configuration_generator.rb

```diff
@@ -282,8 +292,10 @@ def get_vms_definition(node_params)
   # @param node_params [Hash] list of the node parameters
   # @return [String] node definition for the configuration file.
   def generate_node_definition(node_params)
-    tags = { hostname: Socket.gethostname, username: Etc.getlogin,
-             full_config_path: File.expand_path(@configuration_path), machinename: node_params[:name] }
+    tags = @configuration_tags.merge(hostname: Socket.gethostname,
+                                     username: Etc.getlogin,
+                                     machinename: node_params[:name],
+                                     full_config_path: File.expand_path(@configuration_path))
```

Comment:
The value inside the `@conifguration_path` has already been expanded in the setup_command method.